.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,401 @@
+name: E2E Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: Run the full E2E test suite on a specific commit
+        required: true
+  workflow_run:
```

Comment:
I ended up triggering the workflow via the `workflow_run` event rather than manually triggering in CI, because we aren't executing the [Get app entries](https://github.com/department-of-veterans-affairs/vets-website/blob/master/.github/workflows/continuous-integration.yml#L50) step in `master` yet.

The workflow will be cancelled prematurely if the changes in the commit wouldn't cause a single/grouped app build